src/python/move_emojis/code_point.py

```diff
@@ -0,0 +1,28 @@
+def to_emoji(code_point: str) -> str:
+    """
+    Convert a code_point to a character.
+    Example:
+    >>> str_to_char('1F600')
+    '😀'
+    """
+    return chr(int(code_point, 16))
+
+
+def to_bytes(code_point: str) -> bytes:
+    """
+    Convert a code_point to a byte array.
+    Example:
```

Comment:
Nice, are we using `doctest` in CI for this too?

Reply:
Not yet, eventually though!

Reply:
Added to ECO-1558